hadoop-hdds/interface-server/src/main/proto/ScmServerDatanodeHeartbeatProtocol.proto

```diff
@@ -92,6 +101,7 @@ message SCMRegisterRequestProto {
   required NodeReportProto nodeReport = 2;
   required ContainerReportsProto containerReport = 3;
   required PipelineReportsProto pipelineReports = 4;
+  optional HDDSLayoutVersionProto dataNodeLayoutVersion = 5;
```

Comment:
Why not this be a required field? when Datanode firstly register on SCM, it should have its own HDDS layout version.

Reply:
Datanode need not pass the HDDSLaoutVersionProto information during registration.

Reply:
All new fields added should be optional for backward compatibility.